subsys/partition_manager/pm.yml.mcuboot

```diff
@@ -0,0 +1,10 @@
+#include <autoconf.h>
+
+mcuboot_sram:
+  size: CONFIG_PM_SRAM_SIZE
+  address: CONFIG_PM_SRAM_ADDRESS
+#ifdef CONFIG_MCUBOOT_USE_ALL_AVAILABLE_RAM
+  span: [sram_primary, sram_nonsecure, sram_secure]
```

Comment:
Doesn't `sram_primary` already span the entire SRAM range? That is, here we might only need to span that partition.

Reply:
No, on non-secure boards `sram_primary` only spans what is left after sram_nonsecure. Where sram_nonsecure is a partition containing stuff like nrf-modem's RAM partition.

If you think this is not the way to do it I'll try to change my approach to modify `partition_manager` scripts to set `sram_primary` to span the whole RAM region. This will however also require changes in how we partition out RAM for nrf-modem then.

Reply:
I understand. Then I suggest that you stick with the solution you already have here.